www/source/partials/docs/_reference-environment-vars.html.md.erb

```diff
@@ -7,6 +7,7 @@ This is a list of all environment variables that can be used to modify the opera
 | `HAB_AUTH_TOKEN` | build system | no default | Authorization token used to perform privileged operations against the depot, e.g. uploading packages or keys.
 | `HAB_BINLINK_DIR` | build system | `/hab/bin` | Allows you to change the target directory for the symlink created when you run `hab pkg binlink`. The default value is already included in the `$PATH` variable inside the Studio. |
 | `HAB_CACHE_KEY_PATH` | build system, Supervisor | `/hab/cache/keys` if running as root; `$HOME/.hab/cache/keys` if running as non-root | Cache directory for origin signing keys |
+| `HAB_CTL_SECRET` | Supervisor | no default | Shared secret used for communicating with a [remote Supervisor](/docs/using-habitat/#remote-control). |
```

Comment:
This isn't just for remote. By default `hab sup` commands will look for this in `/hab/sup/default/CTL_SECRET`, but if it's not there (this can happen with studios and other overrides), it'll need to be set explicitly.

Also, should https://www.habitat.sh/docs/using-habitat/#remote-control work on live? It looks like that anchor is missing.

Reply:
@baumanj Ok thanks, I'll adjust.

There's [an open PR](https://github.com/habitat-sh/habitat/pull/5240) to make that remote-control link, so I'll merge this after that.